
/// Catches any throws and switches if to af failure after printing the error.
public func printError(_ error: FaroError) {
    switch error {
    case .general:
        print("ðŸ’£ General service error")
    case .invalidUrl(let url):
        print("ðŸ’£invalid url: \(url)")
    case .invalidResponseData(_):
        print("ðŸ’£ Invalid response data")
    case .invalidAuthentication:
        print("ðŸ’£ Invalid authentication")
    case .shouldOverride:
        print("ðŸ’£ You should override this method")
    case .nonFaroError(let nonFaroError):
        print("ðŸ’£ Error from service: \(nonFaroError)")
    }
}
